torch_xla/distributed/spmd/xla_sharding.py

```diff
@@ -495,10 +503,10 @@ def enable_manual_sharding(t: Union[torch.Tensor, XLAShardedTensor],
 
 
 def disable_manual_sharding(t: Union[torch.Tensor, XLAShardedTensor],
-                            partition_spec: Tuple[Union[Tuple, int, str, None]],
+                            partition_spec: PartitionSpec,
                             full_shape: torch.Size,
                             *,
-                            mesh: Mesh = None) -> XLAShardedTensor:
+                            mesh: Optional[Mesh] = None) -> XLAShardedTensor:
   """
   This API disables manual sharding for the given tensor. The partition_spec and full_shape are used to construct the
   output tensor as if the input tensor has not been manual sharded.
```

Comment:
... as if the input tensor has not been manually sharded.

Reply:
done